src/main/java/com/google/sps/data/EmptyListException.java

```diff
@@ -4,6 +4,8 @@
  * A Custom Exception class for an Empty List Exception. 
  */
 public class EmptyListException extends Exception {
+    private static final long serialVersionUID = 6243104809253949369L;
```

Comment:
what is this number used for / where is it from?

Reply:
It was to get rid of a warning that wanted it to have a unique id, the number was generated randomly by a suggested fix